src/ops/imported_ops.jl

```diff
@@ -2,6 +2,7 @@
 
 module Ops
 import TensorFlow
+import SpecialFunctions
```

Comment:
You shouldn't directly edit this file BTW.
It is automatically generated, 

But this change does nothing anyway, so it can happily be removed.